FakebookNotifications/FakebookNotifications.Testing/IntegrationTests/NotificationRepoTests.cs

```diff
@@ -132,12 +132,29 @@ public async Task DeleteNotification_RepoTest()
             //Create repo to work with
             NotificationsRepo repo = new(context);
 
+            //Notification to create
+            Domain.Models.Notification notification = new()
+            {
+                Id = "42069",
+                Type = new KeyValuePair<string, int>("Follow", 12345),
+                LoggedInUserId = "ryan@gmail.com",
+                TriggerUserId = "antonio@gmail.com",
+                HasBeenRead = true,
+                Date = DateTime.Now
+            };
+
+            //Act
+            var result = await repo.CreateNotificationAsync(notification);
+
+            //Assert
+            Assert.True(result);
+
```

Comment:
why are we adding this? at a glance i guess this is an improvement on the existing test... not sure how it was passing before. at least change the test name and the arrange/act/assert comments to reflect what's actually going on.

these tests should be refactored eventually

Reply:
I removed the unnecessary check.